### PR TITLE
Fix 3744

### PR DESCRIPTION
**Pull request by nakai-omer**
## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | #3744 |
| Primary OS tested on | Ubuntu |
| Robotic platform tested on | Custom robot |

---

## Description of contribution in a few bullet points
- Make collision montior's default params fit better the nav2 stack by changing the output cmd topic from cmd_vel to cmd_vel_nav

## Description of documentation updates required from your changes
Need to update `https://navigation.ros.org/tutorials/docs/using_collision_monitor.html` 
Will do in a separate PR to the relevant repo.

#### For Maintainers: 
- [ ] Check that any new parameters added are updated in navigation.ros.org
- [ ] Check that any significant change is added to the migration guide
- [ ] Check that any new features **OR** changes to existing behaviors are reflected in the tuning guide
- [ ] Check that any new functions have Doxygen added
- [ ] Check that any new features have test coverage
- [ ] Check that any new plugins is added to the plugins page
- [ ] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

**Comment by nakai-omer** [comment]
I am not sure this is the correct fix, but anyone that will need to integrate collision monitor with nav2_bringup, which I presume is the recommended way to launch nav2, will have an easier time, with less config tweaking needed.

**Comment by nakai-omer** [comment]
Closing as solution was documentation only at the end

**Comment by nakai-omer** [comment]
Updated for using the smoother by default, just like the default flow in nav2_bringup

File: nav2_collision_monitor/params/collision_monitor_params.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ collision_monitor:
   ros__parameters:
     base_frame_id: "base_footprint"
     odom_frame_id: "odom"
-    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_in_topic: "cmd_vel_smoothed"
```
Comment (SteveMacenski) [inline nav2_collision_monitor/params/collision_monitor_params.yaml:5]: What about for other param files (e.g. any other yaml files in this package from the collision detection node merge recently? + the yaml files in the `nav2_bringup` package)

Comment (nakai-omer) [reply nav2_collision_monitor/params/collision_monitor_params.yaml:5]: @SteveMacenski I couldn't find any other place setting this param in the codebase, or any yaml file setting the collision monitor params. There is the test file `navigation2/nav2_collision_monitor/test/collision_monitor_node_test.cpp`, but I think the value currently there `cmd_vel_in` is the expected one. .

**Review by AlexeyMerzlyakov — COMMENTED** [review]
Please check and sync-up `nav2_collision_monitor/README.md` with latest documentation changes. Otherwise, this change should be enough for CM node.

**Comment by SteveMacenski** [comment]
Yeah, make sure to pull in `main`

**Comment by SteveMacenski** [comment]
@nakai-omer if you had a few minutes, we could finish this up!

**Comment by nakai-omer** [comment]
@AlexeyMerzlyakov Do you think the design image in nav2_collision_monitor/README.md should be updated to include the velocity smoother? Or it comes to show that cmd_vel is rooted in the controller server?

Secondly, there is a link to the configuration guide, that links to https://navigation.ros.org/configuration/packages/configuring-collision-monitor.html, you think it should link directly to https://navigation.ros.org/configuration/packages/collision_monitor/configuring-collision-monitor-node.html?

**Comment by SteveMacenski** [comment]
I think those are fine

**Comment by nakai-omer** [comment]
@SteveMacenski Thanks a lot, I have pulled from main.

**Pull request merged**